_config.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+if(!defined('SITE_BANNER_DIR')) {
+    define('SITE_BANNER_DIR', basename(__DIR__));
```

Comment:
This should use https://docs.silverstripe.org/en/4/developer_guides/templates/requirements/#direct-resource-urls

Reply:
this PR is going to the 1.0 branch for SS3, so will leave this for the SS4 PR